README.md

```diff
@@ -13,63 +13,80 @@ opam install tensorflow
 The opam package starting from version 0.0.8 requires the version 1.0 of the TensorFlow library to be installed on your system under the name `libtensorflow.so`.
 Two possible ways to obtain it are:
 
-* You can build this library from source by following these steps:
-    1. [Install the Bazel build system](http://bazel.io/docs/install.html)
-    1. Clone the TensorFlow repo.
+* __Build the library from source__. Perform the following steps:
+    1. Install the [Bazel build system](http://bazel.io/docs/install.html);
+    1. Clone the TensorFlow repo:
+
+        `git clone --recurse-submodules -b r1.0 https://github.com/tensorflow/tensorflow`
+    1. Configure the build (you will be asked if you want to enable CUDA support):
     
-      `git clone --recurse-submodules -b r1.0 https://github.com/tensorflow/tensorflow`
-    1. In the `tensorflow` directory run `./configure` (you will be asked if you want to enable CUDA support).
-    1. Run `bazel build -c opt tensorflow:libtensorflow.so`.
-    1. The resulting library should then be in `bazel-bin/tensorflow/libtensorflow.so`.
-* You can download prebuilt binaries from Google:
-The releases are available in URLs of the form: `https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-TYPE-OS-ARCH-VERSION.tar.gz`. For example:
+        ```
+        cd tensorflow/
+        ./configure
+        ```
+    1. Compile the library:
+
+       `bazel build -c opt tensorflow:libtensorflow.so`
+       
+       The binary should appear under `bazel-bin/tensorflow/libtensorflow.so`;
+    1. Install the library system-wise, or add it to the environment variables `LIBRARY_PATH` and `LD_LIBRARY_PATH`:
```

Comment:
It seems odd to only specify this when building from source, this should also apply when downloading prebuilt binaries.